setup.py

```diff
@@ -61,6 +62,6 @@ def run(self):
     packages=bbc1_packages,
     scripts=bbc1_commands,
     install_requires=bbc1_requires,
-    data_files = [("/bbc1/common", ["bbc1/common/libbbcsig.so"])],
+    data_files = [("bbc1/common", ["bbc1/common/libbbcsig.so"])],
```

Comment:
It seems that installing in Linux(Utuntu 17.04) is failed. It's succeeded only on Mac.

Reply:
Thanks for checking. Is there any good way to support both Mac and Linux?

Reply:
I think, a way is putting a filecopy script into Python package. 
A filecopy script set libbbcsig.so to environment-dependent directory. 
I don’t think this is a good way. 
But, that's all I can come up with now. 
 I will throw a pull request about this.

Reply:
I'm sorry. 
I didn't check on Linux.
I will close this PR.

Thank you.

Reply:
Thank you for contributing, y-sira!
We would appreciate it if you could follow the project from now on ;)